versions/ufo3/lib.plist.md

```diff
@@ -97,6 +97,28 @@ The handling of the feature file is undefined.
 
 This data is optional. Glyph names must not occur more than once. The list may contain glyphs that are not in the font. An empty list or the absence of this key means that all glyphs are to be exported as-is with groups and kerning untouched.
 
+#### public.unicodeVariationSequences
+
+This key is a dictionary for Unicode Variation Sequences. Keys are *Variation Selector* code points as a hexadecimal numbers and values are dictionaries. These dictionaries contain code points as hexidecimal numbers as keys and glyph names as values. This data can be used to generate the [OpenType cmap subtable format 14] for Unicode [Variation Sequences].
+
+This data is optional.
+
```

Comment:
Would be good to note that this format may be deprecated/changed in UFO4 based on whatever is decided to do about unicodes

Reply:
I'm not sure we should have any note about UFO4 which is still very much undefined, here or anywhere else.

Reply:
Fair point. Ok.